site/app/data/codex_of_abyssiae.json

```
@@ -0,0 +1,231 @@
+{
+  "codex_id": "liber-arcanae/codex_of_abyssiae",
+  "deck_name": "Liber Arcanae — Codex of Abyssiae",
+  "version": "1.1.0",
+  "source_of_truth": "canonical",
+  "notes": "This manifest is identical in circuitum99 and liber-arcanae. It declares the canonical IDs and titles for the 78 cards and provides legacy compat.",
```

Biome (2.1.2)

[error] The key notes was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)